web_src/js/index.js

```diff
@@ -2612,6 +2612,14 @@ $(document).ready(async () => {
     }
   });
 
+  $('form.ui.form button').click(function () {
+    const $this = $(this);
+    $this.addClass('loading');
+    setTimeout(() => {
+      $this.removeClass('loading');
+    }, 1000 * 5); // 5 seconds
```

Comment:
I believe this only applies to "real" forms, in which case receiving a response means the page is reloading I think?

This timeout is a fallback in case it fails so the button isn't stuck perpetually loading.